test/integration/tas/tas_test.go

```diff
@@ -581,6 +581,10 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 					}, util.Timeout, util.Interval).Should(gomega.Succeed())
 				})
 
+				ginkgo.By("trigger topology deletion", func() {
```

Comment:
Does the test repro the issue reliably? 
I think it might be better to split the test case (It) into two:
- the existing one, and 
- the new one with  `trigger topology deletion`, but not `remove topology finalizers to allow deletion`
WDYT?

Reply:
@mykysha FYI I would like to include the fix in the 0.10.1 release I'm planning this week, probably thursday or Friday. Let me know if you cannot address the comment by then.